pyartifactory/utils.py

```diff
@@ -0,0 +1,9 @@
+from pydantic import SecretStr
+from pydantic.json import pydantic_encoder
+
+
+def custom_encoder(obj):
+    if isinstance(obj, SecretStr):
+        return obj.get_secret_value()
+    else:
+        return pydantic_encoder(obj)

```

Comment:
It seems that you haven't run the `pre-commits`. If you had, you could have seen the following errors:
```
pyartifactory/utils.py:1:0: C0114: Missing module docstring (missing-module-docstring)
pyartifactory/utils.py:5:0: C0116: Missing function or method docstring (missing-function-docstring)
pyartifactory/utils.py:6:4: R1705: Unnecessary "else" after "return" (no-else-return)
```

Can be fixed with:
```
"""
Definition of all utils.
"""

from typing import Any

from pydantic import SecretStr
from pydantic.json import pydantic_encoder


def custom_encoder(obj: Any) -> Any:
    """
    Custom encoder function to be passed to the default argument of json.dumps()
    :param obj: A pydantic object
    :return: An encoded pydantic object
    """
    if isinstance(obj, SecretStr):
        return obj.get_secret_value()
    return pydantic_encoder(obj)
```